### PR TITLE
add read.md

### PR DESCRIPTION
**Pull request by Nadja2310**

### CODE REVIEW

File: homework05/read.md [hunk 1]
```diff
@@ -0,0 +1 @@
+hello world!
```
Comment (Nadja2310) [inline homework05/read.md]: Hello

**Review by aliaskov — COMMENTED** [review]
comment

File: homework05/read.md (hunk 1 above)
Comment (AndrisGladinovs) [inline homework05/read.md]: Everything is ok!

Comment (aliaskov) [inline homework05/read.md]: Capitalized

**Review by aliaskov — COMMENTED** [review]
commented

**Review by enextus — COMMENTED** [review]
Interessent :)

**Review by Anastasiaivanov — COMMENTED** [review]
Super!

**Pull request merged**